### PR TITLE
Added secondary positive moon deletion scenario and updated moon crea…

### PR DESCRIPTION
**Pull request by AlexMendelsohnRevature**
Worked a bit on that secondary scenario and fixed the image source path in moon creation so it can be found on anyone's computer.

### CODE REVIEW

**Comment by AlexMendelsohnRevature** [comment]
Updated further to add 1 of my unique scenarios and both negative scenario's for moon deletion

File: src/test/java/com/revature/step/MoonDeletionSteps.java
Comment (justin951) [inline src/test/java/com/revature/step/MoonDeletionSteps.java:1]: alerted not altered

File: src/test/resources/features/MoonDeletionFunctionality.feature
Comment (justin951) [inline src/test/resources/features/MoonDeletionFunctionality.feature:1]: scenario outline names include action 'moon delete'

File: src/test/java/com/revature/step/MoonDeletionSteps.java
Comment (justin951) [inline src/test/java/com/revature/step/MoonDeletionSteps.java:1]: naming convention

**Review by justin951 — CHANGES_REQUESTED** [review]
changes requested as commented

**Comment by justin951** [comment]
resubmit for consolidated changes

**Pull request closed without merging**